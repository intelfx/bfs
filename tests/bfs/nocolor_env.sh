NO_COLOR=1 bfs_pty rainbow >"$OUT"
sort_output
diff_output
