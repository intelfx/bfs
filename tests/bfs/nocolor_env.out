rainbow
rainbow/[1m
rainbow/[1m/[0m
rainbow/broken
rainbow/chardev_link
rainbow/exec.sh
rainbow/file.dat
rainbow/file.txt
rainbow/link.txt
rainbow/lower.gz
rainbow/lower.tar
rainbow/lower.tar.gz
rainbow/lu.tar.GZ
rainbow/mh1
rainbow/mh2
rainbow/ow
rainbow/pipe
rainbow/sgid
rainbow/socket
rainbow/sticky
rainbow/sticky_ow
rainbow/sugid
rainbow/suid
rainbow/ul.TAR.gz
rainbow/upper.GZ
rainbow/upper.TAR
rainbow/upper.TAR.GZ
